<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opsWidget</class>
 <widget class="QWidget" name="opsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>788</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="opLayout">
   <item>
    <widget class="QGroupBox" name="operationSelect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Operation</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>Operation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="opCombo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Shell</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Insert</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Section</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Audio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Compose Demos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Resize</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Move region</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bulk rename</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Add pacing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Animate scroll steps</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Apply to:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="allDemoCheck">
                <property name="maximumSize">
                 <size>
                  <width>10000</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Apply to all demos</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="allStepsCheck">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Apply to all steps</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="matchSubstringCheck">
              <property name="text">
               <string>Apply to steps with matching substring:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Steps containing:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="matchSubstringText">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Demo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="demoTargetCombo"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="opsParamsStack">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="shellTab">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="verticalSpacing">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>11</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="shellImgPath">
              <property name="statusTip">
               <string>Filepath of image to be used as shell for demo assets</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shellBrowseImgBtn">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Coordinates of assets on shell</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shellFgX">
              <property name="maximum">
               <number>100000000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shellFgY">
              <property name="maximum">
               <number>100000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Dimensions of assets on shell</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Width</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shellFgW">
              <property name="maximum">
               <number>100000000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Height</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shellFgH">
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="insertTab">
         <layout class="QFormLayout" name="formLayout_3">
          <property name="verticalSpacing">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>40</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Image to paste</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLineEdit" name="insertImgPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="insertBrowseImgBtn">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Coords of image on assets</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="insertFgX">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="insertFgY">
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Dimensions of image on assets</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>Width</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="insertFgW">
              <property name="baseSize">
               <size>
                <width>1920</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>1920</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>Height</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="insertFgH">
              <property name="maximum">
               <number>1000000000</number>
              </property>
              <property name="value">
               <number>1080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sectionTab">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="1" column="1">
           <widget class="QListWidget" name="sectionRulesListWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Step has TP, next step without TP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step has TP, next step with TP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step has TP, previous step without TP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step has TP, previous step with TP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step has fade-in</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step has jump box</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step TP contains text pattern...</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_10">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Insert new section on...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="sectionCoverageLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Covers 0 steps in selected sections/steps</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_11">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>May not match number of audio soundbites (audio not imported)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="audioTab">
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Mixed section and step audio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Section audio only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Step audio only</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Audio attachment behavior</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="cropTab">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="verticalSpacing">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>40</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Crop start coordinates</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_21"/>
            </item>
            <item>
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_22"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Crop dimensions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>Width</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_25"/>
            </item>
            <item>
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>Height</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_26"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="opsParamsTabsPage6"/>
        <widget class="QWidget" name="opsParamsTabsPage7"/>
        <widget class="QWidget" name="opsParamsTabsPage8"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="applyToBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>175</height>
      </size>
     </property>
     <property name="title">
      <string>Apply to</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="applyToTreeWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Section</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Steps</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="resetStepParamsBtn">
       <property name="text">
        <string>Reset to default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveStepParamsBtn">
       <property name="text">
        <string>Save step parameters</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>opCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>opsParamsStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
